### ISSUE TITLE
Feature request: Disable Input

### ISSUE THREAD

**Issue by MaximilianKresse** [phingofficial/phing#1003]
I want to integrate phing into my Teamcity build process.

Is there is simple way to disable the input? (e.g. in composer there is a "--no-interaction" option)

As far as I can see the only way is to write a own class which implements InputHandler and registers nothing like:
```php
class NoInteractionInputHandler implements InputHandler
{
    public function handleInput(InputRequest $request)
    {
    }
}
```
And call phing with the argument "-inputhandler NoInteractionInputHandler". 

It would be cool if there would be a simple option or a preconfigured InputHandler.

**Comment by mrook** [comment]
Hi @MaximilianKresse, whether phing asks for input is typically dependent on the build file, so I'm not entirely sure this feature makes sense.

**Comment by MaximilianKresse** [comment]
Yeah thats right but I want to be sure that my build process will not run forever if there is a prompt.

Would it hurt to have a "NoInteractionInputHandler"? There isn't much code to care about ;-)

**Comment by mrook** [comment]
Fair enough! No it wouldn't hurt that much ;-) Pull requests welcome!

**Comment by siad007** [comment]
Merged in #1004
